tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from app import app
+
+
+@pytest.fixture()
+def flask_test_client():
+    """
+    Creates the test client we will be using to test the responses
+    from our app, this is a test fixture.
+    :return: A flask test client.
+    """
+    with app.test_client() as test_client:
+        yield test_client
```

Comment:
Are your precommit hooks installed? This file isn't pep8 compliant, which is a nitpick but can lead to inconsistent styling over time within the dev team.

Reply:
if helpful theres also a plugin for on the go formatting, i now have the following set up in VScode user settings:
    "python.formatting.blackArgs": ["--line-length", "120"], // 120 may not be in line with pre commit config
    "python.formatting.provider": "black"

Reply:
One can also add black to PyCharm - please see the following [link](https://black.readthedocs.io/en/stable/integrations/editors.html) . As expected, VSCode is more elegant than Pycharm.